src/debugger/include/debugger/debugger.h

```diff
@@ -1,13 +1,14 @@
 /**
  * @file debugger.h
- * Declares the Debugger Class
+ * Declares the DebuggerCore Class
  */
 
 #pragma once
 
 #include "cartridge/cartridge.h"
 #include "cpu/cpu.h"
 #include "cpu/utils.h"
+#include "debugger/cli_debugger.h"
```

Comment:
Shouldn't be necessary

Reply:
Yup, included the header in main.cpp instead